src/plugins/node-workspace.ts

```diff
@@ -130,16 +133,23 @@ export class NodeWorkspace extends WorkspacePlugin<Package> {
           packagePath,
           this.targetBranch
         );
-        packagesByPath.set(path, new Package(contents.parsedContent, path));
+        const packageJson: ParsedPackageJson = JSON.parse(
+          contents.parsedContent
+        );
+        const pkg: Package = {
```

Comment:
If you made this a class rather than just an interface, it would make the refactor easier to read.

Reply:
Discussed in chat, no need for this change.